libsrc/allegro/src/unix/ufile.c

```diff
@@ -48,6 +48,9 @@
 
 #define PREFIX_I "al-unix INFO: "
 
+#ifndef _WIN32
```

Comment:
I don't think _WIN32 is necessary, this is unix/ufile.c, not supposed to be included on win.

Reply:
Right, I will remove it!